### DISCUSSION TITLE
How can fix NativeAot will trim remove font files

### DISCUSSION THREAD

**Discussion by imJack6** [AvaloniaUI/Avalonia#12376] [Q&A]
Hello, I used FontManagerOptions to customize the font, but after starting NativeAot and PublishTrimmed, I ran the published program and found that the customized font has become the default font. Is there any way to solve it?

**Comment by maxkatz6** [comment]
It will trim any assembly without referenced code in it.
With Inter font we did something like this https://github.com/AvaloniaUI/Avalonia/tree/master/src/Avalonia.Fonts.Inter
And with referencing .WithInterFont that assembly won't be trimmed
